recipes/net/openssh.yaml

```diff
@@ -26,6 +26,13 @@ buildScript: |
         --with-ssl-dir="${BOB_DEP_PATHS[libs::openssl-dev]}/usr" \
         --without-openssl-header-check
 
+    mkdir -p install/.bob
+
+    # add some daemon users
+    cat >install/.bob/sshd.user-table <<EOF
```

Comment:
The version bump and the daemon user should go into dedicated commits each.

Reply:
done